lib/matplotlib/ticker.py

```diff
@@ -2498,7 +2498,7 @@ def tick_values(self, vmin, vmax):
         #
         # "simple" mode is when the range falls entirely within (-t,
         # t) -- it should just display (vmin, 0, vmax)
```

Comment:
```suggestion
        # "simple" mode is when the range falls entirely within [-t, t]
        #  -- it should just display (vmin, 0, vmax)
```
Square brackets to [denote inclusive interval](https://en.wikipedia.org/wiki/Interval_(mathematics)#Notations_for_intervals).  Also I think it's slightly more readable to have the interval on one line.